addons/binding/org.openhab.binding.avmfritz/README.md

```diff
@@ -55,23 +59,33 @@ If correct credentials are set in the bridge configuration, connected AHA device
 
 * AIN (mandatory), no default (AIN number of device)
 
-## Channels
+
+## Supported Channels
 
 | Channel Type ID | Item Type    | Description  | Available on thing |
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍